io/include/writer.hpp

```diff
@@ -0,0 +1,54 @@
+#pragma once
+
+#include "csv/csv_io.hpp"
+#include "edm/cell.hpp"
+#include "edm/cluster.hpp"
+#include "edm/measurement.hpp"
+#include "edm/spacepoint.hpp"
+#include "tmp_edm.hpp"
+
+namespace traccc {
+
+void write_measurements(
+    size_t event,
+    const traccc::host_measurement_container &measurements_per_event) {
+    traccc::measurement_writer mwriter{
+        std::string("event") + std::to_string(event) + "-measurements.csv"};
```

Comment:
Maybe this could be zero-padded to fixed with (not really important).

Reply:
Yeah I agree with Paul on this one, that would be more consistent with the padding that we have everywhere else (data files, Fatras, etc). Perhaps we could handle this in the `traccc::measurement_writer` instead? Do the zero padding there, and accept just an event number as argument to it?